Docs/Updating.md

```diff
@@ -0,0 +1,58 @@
+Updating Reference Assembly Versions
+===
+
+# Creating a new Target Framework
```

Comment:
This section shows how to modify various files, but it's not clear where is the "build" step to run locally and verify things worked. 
Also it may be good to clarify which output files to expect from running those steps.